README.md

```diff
@@ -14,6 +14,38 @@ CLI to communicate with `crispy-api`.
 
 `crispy-cli --help` Shows all available commands
 
+```
+Usage: crispy-cli [options] [command]
+
+CLI tool to communicate with crispy-api
+
+Options:
+  -V, --version  output the version number
+  -h, --help     output usage information
+
+Commands:
+  migrations|m   Operate with Crispy migrations
+```
+
+### Migrations
+
+`crispy-cli migrations --help` Shows all available migrations commands`
+
+```
+Usage: crispy-cli migrations [options] [command]
+
+Options:
+  -h, --help               output usage information
+
+Commands:
+  up <migrationName>       Runs the specified migration
+  down <migrationName>     Will undo the specified migration
+  publish <migrationName>  Send specified migration to server
+  create <migrationName>   Create new migration
+  sync                     Sync migrations directory with server
```

Comment:
What is the difference between `sync` and `publish`?

Reply:
the intention was to split add particular migration, and sync command (where migrations can be deleted or added). `sync` Command will be good for CI, where you have to add only one command

also, I think `unpublish` command is missed here.
Generally, we will have `publish`, `unpublish` to manage migration, but `migrationName` is required there

Reply:
I think we should talk more about usage scenarios for these commands. When and how often a user would use `sync`, `publish`, `unpublish`. Maybe we will able to merge these two into one (just a raw theory), e.g. if migration name is not passed - sync all? 

We may pull out theses command from the current PR to not block it.

Reply:
Yeah, we might start from `sync` command a then try to add more commands based on scenarios we have.

Basically, we are able to start from `migrate/rollback` command as we definitely this functionality.
then start with `sync` command along with API implementation for it